core/src/validation.jl

```diff
@@ -146,6 +146,17 @@ function sorted_table!(
     return table
 end
 
+function valid_config(config::Config)::Bool
```

Comment:
Now that you have this, can it be removed below?

https://github.com/Deltares/Ribasim/blob/e10f657bb56434881bfd3382ddce921b35cff9f5/core/src/model.jl#L50-L53

Reply:
Right, I forgot, thanks.